app/assets/javascripts/chat.coffee

```diff
@@ -99,9 +99,8 @@ class Chat
         $(@).replaceWith html+orig.slice(idx) if html
 
   norlogize_left: (x) ->
-    r = /((\d{4}-\d{2}-\d{2}) )?(\d{2}:\d{2}:\d{2})/g
-    date = x.innerHTML.replace(r, "$2")
-    time = x.innerHTML.replace(r, "$3")
+    date = /\d{4}-\d{2}-\d{2}/.exec(x.dateTime)
```

Comment:
Oops, Chromium doesn't recognize `x` as an `HTMLTimeElement` but as an `HTMLUknownElement`, I need to fix that :/